taichi_three/geometry.py

```diff
@@ -78,12 +88,26 @@ def idx(self):
 
     @classmethod
     def _var(cls, shape=None):
-        return ti.Vector.var(3, ti.i32, shape)
+        return ti.Vector.var(6, ti.i32, shape)
```

Comment:
Shall we use two vectors for `vertex` and `texture_pos` for clarity?
```suggestion
        return ti.Vector.var(3, ti.i32, shape), ti.Vector.var(3, ti.i32, shape)
```

Reply:
TTYL! Looking forward to collaboration :+1: